docs/reference/platforms/macOS/app.rst

```diff
@@ -16,120 +16,23 @@ A macOS ``.app`` bundle is a collection of directory with a specific layout,
 and with some key metadata. If this structure and metadata exists, macOS treats
```

Comment:
I can't post a comment on an unmodified line, but "collection of" should be removed from the first sentence.